omod/src/main/java/org/openmrs/module/webservices/rest/web/controller/PatientFlagRestController.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.webservices.rest.web.controller;
```

Comment:
We can't use the Web Service Module package for our classes/resources. Have a look [here](https://github.com/openmrs/openmrs-module-appointmentscheduling/blob/master/omod/src/test/java/org/openmrs/module/appointmentscheduling/rest/resource/openmrs1_9/TimeSlotResource1_9Test.java) again to see an appropriate package name for a module class/resource.

Use something like:

```java
package org.openmrs.module.patientflags.rest.controller;
```